### ISSUE TITLE
Build error when using ember-cli-page-object v2.0.0-beta.4

### ISSUE THREAD

**Issue by ginomiglio** [Addepar/ember-classy-page-object#39]
The version of `ember-cli-page-object` required in package.json is `~v2.0.0-beta.3`, which includes the newly-released [`v2.0.0-beta.4`](https://github.com/san650/ember-cli-page-object/releases/tag/v2.0.0-beta.4). However, that version [converts `ember-cli-page-object` to a V2 addon](https://github.com/san650/ember-cli-page-object/pull/576), and we now see this error when trying to build our app:

```
ember-classy-page-object needs to depend on ember-auto-import in order to use ember-cli-page-object
```

Our workaround is to force `ember-cli-page-object` to `v2.0.0-beta.3` using [`yarn resolutions`](https://classic.yarnpkg.com/lang/en/docs/selective-version-resolutions/). But it seems like this addon should probably move `ember-auto-import` to `dependencies` instead of `devDependencies` as recommended in the [`ember-auto-import` documentation](https://github.com/ef4/ember-auto-import#installing-ember-auto-import-in-an-addon).

I'll open a PR but I'm not sure if this is a breaking change for consuming apps/addons, so some additional thought might be required there.

**Comment by ginomiglio** [comment]
Looks like #38 should fix this actually.

**Comment by ginomiglio** [comment]
closing this as #38 has been merged and released as `v0.8.0-beta.2`
